src/celengine/processoctree.cpp

```diff
@@ -0,0 +1,335 @@
+
+#include "star.h"
+#include "deepskyobj.h"
+#include "astrooctree.h"
+#include "processoctree.h"
+
+using namespace Eigen;
+
+static constexpr double MAX_STAR_ORBIT_RADIUS = 1;
+
+bool OctreeProcStats::isSelNode(const OctreeNode *node) const
+{
+    if (selection.getType() != Selection::Type_Star)
+        return false;
+    if (selection.star()->getOctreeNode() == node)
+        return true;
+    return false;
+}
+
+void create5FrustumPlanes(Frustum::PlaneType *frustumPlanes, Vector3d position, Quaternionf orientation, float fovY, float aspectRatio)
+{
+    Vector3f planeNormals[5];
+    Quaternionf rot = orientation.conjugate();
```

Comment:
it looks like i told a shit, please revert a variant with matrix. `rot = orientation.toRotationMatrix();... planeNormals[i] = rot.transpose() * planeNormals[i].normalized();` this may be the cause of visibility bugs

Reply:
Bug remains...